### DISCUSSION TITLE
Hasura with Greenplum DB

### DISCUSSION THREAD

**Discussion by rla0h** [hasura/graphql-engine#10081] [Question]
Hasura can connect with PostgreSQL DB, so I think Greenplum DB can connect with Hasura because Greenplum based on PostgreSQL

is it true? Can you Help me plz.. 

I am sorry my English is not good at..

**Comment by manasag** [comment]
Hi @rla0h, we haven't tried Greenplum DB on our side. But most of the PostgreSQL flavor DBs work fine with Hasura. You can try connecting your DB using postgres URL. Please let us know if you face particular issue while trying this.
